File: OOP Lab Final/lab8.java
import java.util.Scanner;

class Volume {
    public double volumeSphere(double radius) {
        return (4.0 / 3.0) * Math.PI * Math.pow(radius, 3);
    }

    public double volumeCube(double side) {
        return Math.pow(side, 3);
    }

    public double volumeCylinder(double radius, double height) {
        return Math.PI * Math.pow(radius, 2) * height;
    }
}
public class lab8 {
    public static void main(String[] args) {
        Volume cal = new Volume();
        Scanner sc = new Scanner(System.in);
        double radius, side, height;

        System.out.print("Enter the radius of the sphere: ");
        radius = sc.nextDouble();
        System.out.println("Volume of the sphere: " + cal.volumeSphere(radius));

        System.out.print("Enter the side length of the cube: ");
        side = sc.nextDouble();
        System.out.println("Volume of the cube: " + cal.volumeCube(side));

        System.out.print("Enter the height of the cylinder: ");
        height = sc.nextDouble();
        System.out.println("Volume of the cylinder: " + cal.volumeCylinder(radius, height));
        sc.close();
    }
}

File: OOP_C++_JAVA/lab8.cpp
#include <bits/stdc++.h>
using namespace std;
class Volume
{
public:
    double volumeSphere(double radius)
    {
        return (4.0 / 3.0) * M_PI * pow(radius, 3);
    }
    double volumeCube(double side)
    {
        return pow(side, 3);
    }
    double VolumeCylinder(double radius, double height)
    {
        return M_PI * pow(radius, 2) * height;
    }
};
int main()
{
    Volume cal;
    double radius, side, height;
    cout << "Enter the radius of the sphere :  ";
    cin >> radius;
    cout << "Volume of the sphere " << cal.volumeSphere(radius) << endl;
    cout << "Enter the side lenght of the cube : ";
    cin >> side;
    cout << "Volume of the cube " << cal.volumeCube(side) << endl;
    cout << "Enter the height of the Cylinder : ";
    cin >> height;
    cout << "Volume of the cylinder " << cal.VolumeCylinder(radius, height) << endl;
    return 0;
}
